src/metrics_server/connection_metrics.ts

```diff
@@ -114,6 +117,11 @@ export function isValidConnectionMetricsReport(
       return false;
     }
 
+    // Check that `tunnelTime` is a number larger than min tunnel time.
+    if (typeof userReport.tunnelTimeMs !== 'number' || userReport.tunnelTimeMs < MIN_TUNNEL_TIME) {
```

Comment:
It must be ok to not have the Tunnel Time parameter. Even if we forced it in the servers (not sure we want to do that), there's a transition period. Treat it as optional.

Reply:
Ugh yes that's my bad. Fixed.